ftl/cached/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM bazel/reproducible:ubuntu_vanilla
```

Comment:
Why do we need this Dockerfile?

Reply:
we don't, i will remove it in another PR as kokoro is a bit flaky rn